README.md

```diff
@@ -1,19 +1,19 @@
 [![][ButlerImage]][website] 
 
 # About
-In a nutshell, Jenkins CI is the leading open-source continuous integration server. Built with Java, it provides over 1000 plugins to support building and testing virtually any project.
+In a nutshell, Jenkins is the leading open-source automation server. Built with Java, it provides over 1000 plugins to support building and testing virtually any project.
```

Comment:
Sounds like a ci server.
Perhaps "... building, testing, and deploying..." to make it more than just ci?

Or use the text from https://wiki.jenkins-ci.org/display/JENKINS/Meet+Jenkins